src/data/getReviews.py

```diff
@@ -184,6 +185,31 @@ def getReviewDetail(userReviewDriver, reviewDiv, review):
     name = userReviewDiv.find_element_by_xpath('//div[@class="username mo"]/span').text 
     location = userReviewDiv.find_element_by_xpath('//div[@class="location"]/span').text 
 
+    # Gather Category Ratings
+    # This returns a collection of divs that contain both the bubble rating and rating description
+    # I'm iterating through the collection and using the modulus function to alternate between parsing
+    # I'm sure there's a more elegant way to do this
+    ratingElements = userReviewDiv.find_elements_by_xpath('//div[@id="review_' + review.Id + '"]//li[@class="recommend-answer"]/div')
+    starRatings = []
+    ratingCategories = []
+    x = 0
+    for ratingElement in ratingElements:
```

Comment:
Suggestion for future use: I love using enumerate() for loop counting.  You can just call:
`for x, ratingElement in enumerate(ratingElements):` and it will handle the initialization of x as well as the increments.  Maybe a handy Python tool for your toolbelt.